src/ankisyncd/sync.py

```diff
@@ -58,15 +73,15 @@ def changes(self):
                  decks=self.getDecks(),
                  tags=self.getTags())
         if self.lnewer:
-            d['conf'] = json.loads(self.col.backend.get_all_config())
+            d['conf'] = self.col.all_config()
             d['crt'] = self.col.crt
         return d
 
     def mergeChanges(self, lchg, rchg):
         # then the other objects
         self.mergeModels(rchg['models'])
         self.mergeDecks(rchg['decks'])
-        self.mergeTags(rchg['tags'])
+        # self.mergeTags(rchg['tags'])
```

Comment:
Minor: Remove commented code. It'll still be available in the Git history if needed.